### PR TITLE
Fix file paths for test arguments

### PR DESCRIPTION
**Pull request by aleksandr-dudko**
Add fix for file paths for test arguments

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Review by andreydevyatkin — APPROVED** [review]
Thanks!

File: .github/workflows/beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml [hunk 1]
```diff
@@ -79,10 +79,10 @@ jobs:
...
           test-language: python
           argument-file-paths: |
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_100b_Single_Key.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_100b_Multiple_Keys.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_10kB.txt
-            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/python_CoGBK_Dataflow_Batch_2MB.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_100b_Single_Key.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_100b_Multiple_Keys.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_10kB.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_CoGBK_Dataflow_Batch_2MB.txt
```
Comment (damccorm) [inline .github/workflows/beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml:85]: What was the issue we're fixing here? Are paths too long?

I like the per directory organization, if file length is the issue maybe we could do something like `${{ github.workspace }}/.github/workflows/load-tests-job-configs/python-cogbk/dataflow_Batch_2MB.txt` which truncates to basically the same length

Comment (aleksandr-dudko) [reply .github/workflows/beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml:85]: Yes, I corrected the file path to the correct one, following the same approach we use for all other jobs.

Comment (damccorm) [reply .github/workflows/beam_LoadTests_Python_CoGBK_Dataflow_Batch.yml:85]: Ok, I think it makes sense to attach to the consistent pattern, so I'll merge. In the future we may consider nesting these hierarchically

**Pull request merged**